src/main/java/pl/cleankod/exchange/core/usecase/FindAccountAndConvertCurrencyUseCase.java

```diff
@@ -34,7 +34,7 @@ public Optional<Account> execute(Account.Number number, Currency targetCurrency)
 
     private Money convert(Money money, Currency targetCurrency) {
         if (!baseCurrency.equals(targetCurrency)) {
-            return money.convert(currencyConversionService, targetCurrency);
+            return currencyConversionService.convert(money, targetCurrency);
```

Comment:
Why?

Reply:
Why do we use a domain pojo in between to pass a conversion service as an argument there.. Makes no sense to me so I've refactored the redundancy.